<launch>
  <node name="manipulator" pkg="manipulator" type="manipulator_node.py" output="screen" />
  <node name="main_state" pkg="main_state" type="main.py" output="screen" args="BasicFunctional" />
  <node ns="door" name="door_detection" pkg="door_detection" type="door_detection_node">
    <remap from="/scan" to="/laser/scan"/>    
  </node>

</launch>
